homeassistant/components/switch/rpi_rf.py

```diff
@@ -27,8 +27,10 @@
 DEFAULT_SIGNAL_REPETITIONS = 10
 
 SWITCH_SCHEMA = vol.Schema({
-    vol.Required(CONF_CODE_OFF): cv.positive_int,
-    vol.Required(CONF_CODE_ON): cv.positive_int,
+    vol.Required(CONF_CODE_OFF):
```

Comment:
I think the regex solution should work as well, doesn't it? Or is there a reason for me to change it to a list?